common/src/main/java/tachyon/network/protocol/RPCErrorResponse.java

```diff
@@ -44,17 +40,25 @@ public static RPCErrorResponse decode(ByteBuf in) {
   }
 
   @Override
-  public int getEncodedLength() {
-    // 1 short (mStatus)
-    return Shorts.BYTES;
+  public void encode(ByteBuf out) {
+    out.writeShort(mStatus.getId());
   }
 
   @Override
-  public void encode(ByteBuf out) {
-    out.writeShort(mStatus.getId());
+  public int getEncodedLength() {
+    // 1 short (mStatus)
+    return Shorts.BYTES;
   }
 
   public Status getStatus() {
     return mStatus;
   }
+
+  public Type getType() {
+    return Type.RPC_ERROR_RESPONSE;
+  }
+
+  public String toString() {
+    return mStatus.getMessage();
```

Comment:
I was thinking this should not be the status message, but the string representation of the rpc message itself. So, maybe the result should look like:

RPCErrorResponse(SUCCESS)